libraries/Microsoft.Bot.Builder.LanguageGeneration/Evaluator.cs

```diff
@@ -436,7 +443,7 @@ private object EvalExpression(string exp, ParserRuleContext context = null, stri
 
                 CheckExpressionResult(exp, error, result, templateName, context, errorPrefix);
             }
-            else if (result == null && !strictMode)
+            else if (result == null && lgOptions.StrictMode == false)
```

Comment:
strict mode is a nullable property, so, maybe you should use 
`lgOptions.Strict != true` replace `lgOptions.StrictMode == false`